set.mm

```diff
@@ -60102,8 +60105,8 @@ in the range of the function (the implication "to the right" is always
       H $.
   $}
 
-  $( The Cartesian product of two singletons.  (Contributed by Mario Carneiro,
-     30-Apr-2015.) $)
+  $( The Cartesian product of two singletons is the singleton consisting in the
+     associated ordered pair.  (Contributed by Mario Carneiro, 30-Apr-2015.) $)
```

Comment:
consisting in -> consisting of

Reply:
Both are correct, but since a singleton has by definition a single element, that element really determines it, and this is what "consist in" emphasizes.

Reply:
"consist in" wasn't familiar to me, but the dictionary (at least https://www.merriam-webster.com/dictionary/consist%20in ) agrees with @benjub - "consist in" refers to "essential or main part" and "consist of" is merely "made up of".

Reply:
OK, again something learned.